### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by HStakhiv**
image
image
image
image
image
image
image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+*.idea
+*.sqlite3
+*.pyc
```
Comment (HStakhiv) [inline .gitignore:3]: Why weren't these files ignored?

Comment (HStakhiv) [reply .gitignore:3]: Oh. I must untrack the file before adding a rule to ignore it :(

**Review by shurushku — APPROVED** [review]
Looks ok

File: taxi/models.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        verbose_name_plural = "drivers"
+
+    def __str__(self):
+        return f"{self.username} ({self.first_name} {self.last_name})"
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(
+        to=Manufacturer, on_delete=models.CASCADE, related_name="manufacturers"
```
Comment (Y-Havryliv) [inline taxi/models.py]: remind yourself how to name `related_name` properly
https://github.com/mate-academy/py-movie-moviesession-cinemahall/blob/master/checklist.md#code-efficiency

File: taxi/models.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    )
+    drivers = models.ManyToManyField(
+        to=settings.AUTH_USER_MODEL, related_name="drivers"
```
Comment (Y-Havryliv) [inline taxi/models.py]: same here

**Review by Y-Havryliv — CHANGES_REQUESTED** [review]

File: taxi/models.py (hunk 2 above)
Comment (HStakhiv) [reply taxi/models.py]: I saw that you requested changes from Yaroslav with the same problem. But it was too late xD
image

**Review by Y-Havryliv — APPROVED** [review]
Nice!

**Pull request closed without merging**